lib/spree/adyen/form.rb

```diff
@@ -132,7 +132,8 @@ def order_params order
             merchant_reference: order.number.to_s,
             country_code: order.billing_address.country.iso,
             payment_amount: (order.total * 100).to_int,
-            shopper_locale: I18n.locale.to_s.gsub("-", "_")
+            shopper_locale: I18n.locale.to_s.gsub("-", "_"),
+            shopper_email: order.email
```

Comment:
Yes, the shopper_email is now required to calculate a valid Merchant Signature. Please hold fire on merging as this PR is still not ready to merge.